docs/api/command.md

```diff
@@ -12,9 +14,27 @@ A command is an object of the following structure:
   type, // A string that contains the command type name.
   aggregateId, // A string that uniquely identifies an aggregate instance.
   aggregateName, // The name of an aggregate that the command targets.
-  payload, // An object of arbitrary structure that contains data attached to the command. (optional)
+  payload, // An object of arbitrary structure that contains data attached to the command. 
   jwt // A JSON Web Token attached to the web request used to send the command. (optional)
```

Comment:
Can we move "optional" to the beginning of the comment?

`jwt // Optional. A JSON Web Token attached to the web request used to send the command.`

Reply:
Moved "Optional" to the beginning here and in all other comments throughout the document.

Reply:
Thank you.